src/http/body.c

```diff
@@ -16,20 +16,19 @@ struct Body {
 	Node* head;
 };
 
-#define SPAIR(node) ((SPair*)node->value)
+#define SPAIR(node) ((SPair*)(node)->value)
 
 // If you want to implement Content-Type please refer RFC 2616 7.1
 
-static inline _Bool is_sepported_type(const char* str) {
-	if(!str)
-		return false;
-	return !strncmp(str, "application/x-www-form-urlencoded", 34);
+static inline _Bool is_supported_type(const char* str) {
+	return !strncmp(str, "application/x-www-form-urlencoded", 33);
```

Comment:
What if str is a NULL ptr?

Reply:
The function will be called if content_type is not null.
Refer to line 29.
I recommend refactor that condition or keep this change.